packages/styles/dark-theme.css

```diff
@@ -0,0 +1,77 @@
+.cauldron--theme-dark,
+.theme-dark {
+  --button-background-color-primary: var(--accent-medium);
+  --button-background-color-primary-disabled: var(--accent-medium);
+  --button-background-color-secondary: var(--accent-medium);
+  --button-background-color-secondary-disabled: var(--accent-medium);
+  --button-background-color-error-disabled: var(--accent-medium);
+  --header-text-color-dark: var(--white);
+
+  --button-background-color-primary-active: #a7e9f7;
+  --button-background-color-secondary-active: var(--accent-light);
+  --button-background-color-error-active: #fea7a6;
+
+  --header-text-color-dark: var(--white);
+}
+
+.cauldron--theme-dark .Layout,
```

Comment:
These styles should be co-located in their own individual css files.

For example, `button` styles should be in `button.css`, layout styles should be in `layout.css` and so on. We've been trying to move each styles to their own component CSS as it's easier to maintain individually as opposed to lumped together in a single file.